client/style.css

```diff
@@ -460,3 +465,103 @@ h1{
 .star-review__star--empty > span::after {
   width: 0;
 }
+
+/* || Carousel and Cards */
+
+
+
+.carousel {
+  display: grid;
+  gap: 2rem;
+  grid-auto-flow: column;
+}
+.carousel__overflow {
```

Comment:
* Underscore: I'm not familiar with this style as much as hyphen-separated class names. Is this the BEM (https://en.bem.info/methodology/naming-convention/ )style? It seems clear enough, we should decide on a consistent approach for all css rules.

* spacing after class selector: I mentioned the similar concern with Griselda's CSS PR as most css I've seen doesn't have a space between the class selector and the braces.

Reply:
I use BEM, but am open for whatever.

I've always kept spaces between the selector and the brace for ease of reading/delineation. Space = new thing. e.g. `div>.thing~section#other-thing{` vs `div > .thing ~ section#other-thing {`

Reply:
Great. We discussed this and decided to adopt a space after class selectors so this looks good.